### ISSUE TITLE
[FEATURE_REQUEST] Bulk Character Upload

### ISSUE THREAD

**Issue by MichaelData** [SillyTavern/SillyTavern#2952]
### Have you searched for similar requests?

Yes

### Is your feature request related to a problem? If so, please describe.

SillyTavern is painfully slow at uploading multiple character. I had SillyTavern break 2 months ago. I had to do a full reinstall. I then tried to upload 7500 character...it took days for it to finish uploading. 

Sometimes it would just break but the command line does not even tell me the name of the file it left off on.

### Describe the solution you'd like

I’d love to see a feature that allows users to create a bulk folder where multiple characters can be stored and uploaded all at once. This would significantly streamline the process of adding large amounts of characters to SillyTavern. Additionally, it would be great if the system could resume from where it left off in case of a disruption, rather than starting over. This would make handling a large number of uploads more efficient and user-friendly.

It would be even better if there was away to have character upload in the background so you can still use SillyTavern while they are uploading. 

### Describe alternatives you've considered

I wanted to just copy and paste the characters into the folder character are loaded into but they don't show up. 

### Priority

Medium (Would be very useful)

### Are you willing to test this on staging/unstable branch if this is implemented?

Yes

**Comment by cloak1505** [comment]
To be clear, do you mean import/export/both?

**Comment by MichaelData** [comment]
> To be clear, do you mean import/export/both?

Yes exporting would be great also.

**Comment by MichaelData** [comment]
> To be clear, do you mean import/export/both?

Yes Importing and Exporting. So yes Both

**Comment by ObscenityIB** [comment]
Adding a comment here, character importing used to work fine, it would import all the characters and then ask about the tags.

Now, since a recent update, characters take around 10 seconds each to import, since it does a flashy fading animation, then pops up two notification boxes with 5 second timers on them.

Sure, it looks cool, but at the cost of reduced function of the interface.

A bulk import feature as suggested would restore it.

**Comment by MichaelData** [comment]
> Adding a comment here, character importing used to work fine, it would import all the characters and then ask about the tags.
> 
> Now, since a recent update, characters take around 10 seconds each to import, since it does a flashy fading animation, then pops up two notification boxes with 5 second timers on them.
> 
> Sure, it looks cool, but at the cost of reduced function of the interface.
> 
> A bulk import feature as suggested would restore it.

Yeah that's why I just want a bulk background character uploader that will just upload the character quick because 10 seconds per character and i'm trying to do a full reinstall imagine if I've got 2000 characters. that becomes over 5 hours to upload them lol.

**Comment by ObscenityIB** [comment]
Can confirm this is fixed now.

Back to blazing fast once again!

**Comment by Wolfsblvt** [comment]
Well, seems like this is fixed and got released then
